aggsender/types/certificate_build_params.go

```diff
@@ -52,6 +54,7 @@ type CertificateBuildParams struct {
 	ToBlock                        uint64
 	Bridges                        []bridgesync.Bridge
 	Claims                         []bridgesync.Claim
+	Unclaims                       map[*big.Int]*bridgesynctypes.Unclaim
```

Comment:
If we only need this to be a map because of the filtering when we convert claims to imported bridge exits, I would say, just make a local map there and make this a list, sorted by log index and block number.

Reply:
I don't see that on the request to the prover. Its a slice in the proto definition: `[]*aggkitProverV1Proto.Unclaim`.